ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.20.yaml

```diff
@@ -539,16 +539,28 @@ tests:
     workflow: openshift-e2e-vsphere
   timeout: 6h0m0s
 - always_run: false
-  as: e2e-aws-ovn-virt
+  as: e2e-metal-ipi-ovn-virt-dualstack
```

Comment:
hmm so we are replacing aws with metal here but on master we keep both? why is that?

Reply:
Good  catch, removing